swagger/swagger.yml

```diff
@@ -0,0 +1,68 @@
+consumes:
+- application/json
+definitions:
```

Comment:
shift these definitions block the bottom of the file for better readability

Reply:
Done, please check